src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -133,4 +133,9 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
             await this._keytar.setPassword(this._serviceName, this._serverId, password);
         }
     }
+
+    public supportsStoredProcedures(): boolean {
+        const version: string | undefined = this.server.version;
+        return !!version && parseFloat(version) >= 11;
```

Comment:
Does postgres use semver? If so, I suggest you use the [semver package](https://www.npmjs.com/package/semver) instead of `parseFloat`. Theortically `parseFloat` would fail for versions like "11.0.0-alpha"